teams.json

```diff
@@ -13,6 +13,24 @@
         "storagetransfer"
       ]
     },
+    {
```

Comment:
I have no idea what the outcome of double counting the APIs like this will be. Likely not deterministic 😆

Reply:
I can remove the old team I suppose?